src/commands/app/deploy.js

```diff
@@ -324,6 +342,10 @@ Deploy.flags = {
     default: true,
     exclusive: ['action']
   }),
+  'force-deploy': flags.boolean({
+    description: '[default: false] Force deploy extensions regardless of published status or workspace',
+    default: false
+  }),
   'force-publish': flags.boolean({
```

Comment:
Not directly part of this PR but I think there might be some confusion with `force-publish` flag unfortunately. Because you cannot publish an app via the CLI but only update the extension registry manifest in Exchange..
We should clarify this in the description, although the best would be to rename the flag.. cc @shazron @purplecabbage

Reply:
Agreed, that flag does sound like it could use a better name. That being said, what can users tangibly accomplish by updating the extension registry manifest in Exchange? What would that be considered as an action other than `publish`?

Reply:
I think the whole publish vocabulary is misleading, what "publish" does here is just update the extension registry metadata. `force-publish` is overwriting all previous extension points config (there can be multiple extension points assigned to a workspace)

This is all confusing I realize now.

Reply:
ok makes sense let's keep  `force-deploy` then. Renaming the flag would be a breaking change. I think we can address the code renaming and flags renaming (`force-publish` but also `(no)-publish`) as part of a separate Issue/PR.